components/wb-urlmapping/selectinput-en.html

```diff
@@ -222,3 +223,251 @@ <h2>Example without source</h2>
 &lt;/fieldset></code></pre>
 	</details>
 </section>
+
+<section id="filteredEventsList3" class="wb-filter wb-tagfilter wb-paginate provisional" data-wb-filter='{
+	"selector": "li",
+	"section": ".events-list3",
+	"uiTemplate": "#searchFilter3"
+}' data-wb-paginate='{
+	"itemsPerPage": 5,
+	"section": ".events-list3",
+	"selector": "li",
+	"uiTarget": "#filteredEventsListPagination3"
+}'>
+	<h2>Example with Tag-filter component</h2>
+	<div id="searchFilter3" class="row mrgn-tp-lg">
```

Comment:
Please either add an instruction or a link on how to test this example. For example: "Add the following parameter in the url..."